docs/docs/api-reference/core/useSetRecoilState.md

```diff
@@ -7,6 +7,37 @@ Returns a setter function for updating the value of writeable Recoil state.
 
 ---
 
-- `state`: writeable Recoil state.
+- `state`: writeable Recoil state (an [`atom`](/docs/api-reference/core/atom) or a _writeable_ [`selector`](/docs/api-reference/core/selector))
+
+This is the recommended hook to use when a component intends to write to state without reading it.
 
 ### Example
+
+```javascript
+import { atom, useSetRecoilState } from 'recoil';
```

Comment:
To standardize with our Prettier config.  We're trying to automate that as part of the tests for code at least.
```
import {atom, useSetRecoilState} from 'recoil';
```

Reply:
I see, yeah automation would be great!